npm/init.js

```diff
@@ -16,15 +17,30 @@ function validateNpmVersion() {
     }
 }
 
+function download(url) {
+    // We detect proxy by looking at the environment variable
+    var agent = https;
+    if(process.env.https_proxy && process.env.https_proxy.length > 0) {
+        agent = http;
+        url = process.env.https_proxy + starturl.replace('https://', '/https/');
+    }
+
+    agent.get(url, function(res) {
+        if(res.statusCode == 301 || res.statusCode == 302) {
+            download(res.headers.location);
+        } else if (res.statusCode == 200) {
+            writeToFile(res)
+        } else {
+            console.log('Unexpected status code during JFrog CLI download')
```

Comment:
I suggest we add the received status code to the error message.

Reply:
done